web/webkit/src/main/resources/toserve/lift.js

```diff
@@ -680,7 +685,7 @@
         success: onSuccess,
         error: function(_, status) {
           if (status != 'abort')
-            return onFailure.apply(this, arguments)
+            return onFailure.apply(this, arguments);
```

Comment:
Codacy flagged this as being better with curlies around the if body, I tend to agree.